source/includes/language-compatibility-table-go.rst

```diff
@@ -0,0 +1,34 @@
+The following compatibility table specifies the minimum version of the
+Go programming language to use with versions of the {+driver-long+}.
+
+The first column lists the driver version.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large
+
+   * - Go Driver Version
+     - Minimum Go Language Version
+   * - 1.10
+     - 1.13
```

Comment:
Since this is the v1.9 branch, should the compat table omit the v1.10 compat?

Reply:
Discussed over slack, agreed all compat tables should include all versions.

Adding shared directive back in as discussed on slack with Chris.